src/main/kotlin/com/devfloor/untitled/article/application/ArticleFacade.kt

```diff
@@ -17,35 +16,51 @@ class ArticleFacade(
     private val articleHashtagService: ArticleHashtagService,
     private val articleFavoriteService: ArticleFavoriteService,
     private val articleOptionService: ArticleOptionService,
-    private val optionService: OptionService,
     private val hashtagService: HashtagService,
+    private val optionService: OptionService,
 ) {
     @Transactional
     fun create(request: ArticleRequest, user: User): Long {
         val article = articleService.create(request.toArticle(request, user))
-        if (request.isOptionsNotEmpty()) {
+        if (request.isOptionsNotEmpty) {
             optionService.showAllByOptionType(request.options)
-                .map { ArticleOption(article, it) }
-                .let { articleOptionService.createAll(it) }
+                .let { articleOptionService.createAll(article, it) }
         }
-        request.hashtags.map { hashtagService.createByName(it) }
+        request.hashtags
+            .map { hashtagService.createByName(it) }
             .map { articleHashtagService.create(ArticleHashtag(article, it)) }
```

Comment:
전달되는 인자를 하나만 받는 함수일 경우 참조로 사용이 가능하더라고. Java에서 함수 참조를 사용하는거랑 비슷한데

```suggestion
        request.hashtags
            .map(hashtagService::createByName)
            .map{ ArticleHashtag(article, it) } 
            .map(articleHashtagService::create)
```

이런 식으로? 이건 그냥 참고해보라고 적어뒀어!